admin/views/tabs/dashboard/dashboard.php

```diff
@@ -14,8 +14,11 @@
  */
 $notifications_data = Yoast_Notifications::get_template_variables();
 
+$notifier = new WPSEO_Configuration_Notifier();
+$notifier->listen();
```

Comment:
Could the `listen();` method be called in the class construct?

Reply:
Spoke with @moorscode and this should not be done since constructors are bad practice for side-effects

Reply:
Or phrased differently, it's bad practice to have side-effects in constructors :)
It makes writing tests a lot harder, and also obscures functionality.